hack/test-end-to-end.sh

```diff
@@ -247,7 +248,7 @@ if [[ "$ROUTER_TESTS_ENABLED" == "true" ]]; then
 
     echo "[INFO] Installing router with master url of ${API_SCHEME}://${CONTAINER_ACCESSIBLE_API_HOST}:${API_PORT} and starting pod..."
     echo "[INFO] To disable router testing set ROUTER_TESTS_ENABLED=false..."
-    "${OS_ROOT}/hack/install-router.sh" "router1" "${API_SCHEME}://${CONTAINER_ACCESSIBLE_API_HOST}:${API_PORT}"
+    OPENSHIFT_CA_DATA=$(cat "${CERT_DIR}/system:openshift-client/root.crt") OPENSHIFT_CERT_DATA=$(cat "${CERT_DIR}/system:openshift-client/cert.crt") OPENSHIFT_KEY_DATA=$(cat "${CERT_DIR}/system:openshift-client/key.key") "${OS_ROOT}/hack/install-router.sh" "router1" "${API_SCHEME}://${CONTAINER_ACCESSIBLE_API_HOST}:${API_PORT}"
```

Comment:
install-router.sh was already passing the actual data along for CA.  I'm not necessarily attached to it, but should we touch it or ask Paul to find all the updates he needs to make?